ui/docs/Architecture.md

```diff
@@ -19,12 +19,11 @@ following:
 ![Starter app topology](./ArchImages/Topology.jpg)
 
 When running the Jar file, a Vertx server will be started, which in turn will
-launch three verticles. One verticle will handle records being produced to
-Kafka, one verticle will handle records being consumed from Kafka, and finally
-the third verticle deployed will server the UI when requested, along with all
-supporting content, such as fonts and images. The producer and consumer 
-verticles are configured to access Kafka via configuration provided by the user
-at start up.
+launch two verticles. One verticle will handle records being produced to
```

Comment:
This still isn't correct. I suggest:
One verticle will handle records being produced to
Kafka, and the second verticle deployed will handle records being consumed from
Kafka and serve the UI 
 when requested, along with all supporting content, such as fonts and images. 
 The application is configured to access Kafka via 
 configuration provided by the user at start up.

Reply:
Ahh - sorry I thought you meant 1 for producing/consuming, and one for UI, not one to produce and one to consume/serve the UI! Will update.